### DISCUSSION TITLE
European User Consent

### DISCUSSION THREAD

**Discussion by Deodes** [invertase/react-native-firebase#5362] [Q&A] [answered]
I'm using react native firebase admob, and wanted to figure out, if i have to follow "European User Consent" guide from docs in my app (is it necessary?). Also wanted to know, how to combine it with App Tracking Transparency (iOS 14+)?

**Comment by mikehardy** [comment, marked as answer]
Hi there! This is not react-native specific, this is a general consent / permissions issue for any iOS app. I think stackoverflow is a better venue and suggest searching + posting there
